### PR TITLE
maintainers: add mikecm

### PR DESCRIPTION
**Pull request by MaxwellDupre**
Please add to maintainers list user mikecm

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -11707,6 +11707,12 @@
     github = "mihnea-s";
     githubId = 43088426;
     name = "Mihnea Stoian";
+  };
+    mikecm = {
```
Comment (bjornfor) [inline maintainers/maintainer-list.nix:11711]: This line is indented 2 spaces too much.

Comment (bjornfor) [reply maintainers/maintainer-list.nix:11711]: And the whole entry must be moved down to be properly sorted (as pointed out by the CI failure).

Comment (MaxwellDupre) [reply maintainers/maintainer-list.nix]: Done and done.

Comment (bjornfor) [reply maintainers/maintainer-list.nix]: And squash commits.

Comment (MaxwellDupre) [reply maintainers/maintainer-list.nix]: How do I do that?
A `git merge --squash` in my clone (on the appropriate branch)?
then push..

Comment (bjornfor) [reply maintainers/maintainer-list.nix]: I usually do `git rebase -i master` and change "pick" lines to "squash" lines.

**Comment by bjornfor** [comment]
Thanks!

**Pull request merged**